### ISSUE TITLE
Razer Hydra Emulation

### ISSUE THREAD

**Issue by andregm3** [cboulay/PSMoveService#122]
is it possible to make it work as Razer Hydra so we could have a lot of games supported to play, i know there is something done with MotionInJoy and FreePIE, would it work with PS Move Service ?

**Comment by cboulay** [comment]
I'll leave this open for a little while so others can comment but this is definitely beyond the scope of PSMoveService.

**Comment by andregm3** [comment]
https://www.youtube.com/watch?v=H6yA-boXNeU

**Comment by andregm3** [comment]
http://www.mtbs3d.com/phpBB/viewtopic.php?f=139&t=19998

**Comment by andregm3** [comment]
https://www.youtube.com/watch?v=hykjEBYzCnE

**Comment by DjDyll** [comment]
Any ideas of the 3d print files are publicly available? That would be awesome either to straight up print, or recreate the model and then print.

**Comment by andregm3** [comment]
![ps](https://cloud.githubusercontent.com/assets/4154732/16906161/e8b016e8-4c88-11e6-90ef-1330f49cfa07.png)

**Comment by andregm3** [comment]
http://www.realovirtual.com/foro/topic/4522/emular-razer-hydra-stem-usando-2-psmove-2-navigation

**Comment by HipsterSloth** [comment]
I think once we had support for the navi (something that is partially implemented now and I just need to finish once the other higher priority work is done), I can just expose the navi buttons to steamvr the same way I expose PSMove buttons. I could extended the button remapping code a bit to associate the navi with a particular move controller, so it would appear to steamvr that the psmove has a thumbstick and other buttons. That way we wouldn't need to emulate the razor hydra or use FreePie. 

That said, if someone still wanted to do use FreePIE they could do that. The psmove client will still expose the navi controller like all of the other controllers. A PSMoveService plugin for FreePie could then remap the navi controller buttons anyway it liked. But as cboulay said, that part is beyond the scope of this project.

**Comment by andregm3** [comment]
Great, i can't barely wait ! and yes i still want Hydra emulation for non steam VR games =]

**Comment by andregm3** [comment]
![4930862644_c78e8b3272_b](https://cloud.githubusercontent.com/assets/4154732/16909603/2da6589a-4caa-11e6-90ce-463566ec2bc6.jpg)

**Comment by cboulay** [comment]
I'm closing this because it isn't really an issue relevant to PSMoveService. Hydra emulation would have to be done elsewhere. Providing custom button mapping might be something we can do but that is a separate issue entirely.

That being said, @andregm3  feel free to make a Wiki entry regarding all the resources and information you've found about using PSMove with the Navi controller.
